shell/platform/darwin/ios/framework/Source/FlutterAppDelegate.mm

```diff
@@ -14,6 +14,7 @@
 static NSString* kUIBackgroundMode = @"UIBackgroundModes";
 static NSString* kRemoteNotificationCapabitiliy = @"remote-notification";
 static NSString* kBackgroundFetchCapatibility = @"fetch";
+static NSString* kRestorationStateAppModificationKey = @"mod-date";
```

Comment:
These should really be `static NSString* const`, sorry I know you didn't set the precedent =(

Reply:
Changed.